<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1607</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aurora - View</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QwtPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
   </widget>
   <widget class="Aurora::QViewFile" name="view2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>532</width>
      <height>835</height>
     </rect>
    </property>
   </widget>
   <widget class="Aurora::QViewFile" name="view1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>532</width>
      <height>835</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="frcPointsSpinBox">
    <property name="geometry">
     <rect>
      <x>1264</x>
      <y>525</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>530</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>FRC - Number of points:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1607</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Aurora::QViewFile</class>
   <extends>QWidget</extends>
   <header>QtSupport/QView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
